objectRecognitionSource/README.md

```diff
@@ -200,6 +200,9 @@ Most of the options required for neuralNet are dependent on the specific impleme
     1.  "local"     - This will save images to the disk (outputDir must be specified in order for this to work).
     2.  "vantiq"    - This will save images as documents in VANTIQ. No images will be saved locally even if outputDir is specified.
     3.  "both"      - This will save the images both to the disk and as documents in VANTIQ. (outputDir must be specified in order to save locally)
+    
+**NOTE:** All of the following options are only necessary if the "saveImage" option has been set.
+
```

Comment:
maybe change "only necessary" to "relevant only" -- since they are all optional, they are never "necessary".  But they are meaningless if ```saveImage``` is not true.